nanoFramework.Graphics/Primitive/Bitmap.cs

```diff
@@ -267,6 +267,17 @@ public void DrawTextInRect(string text, int x, int y, int width, int height, uin
             DrawTextInRect(ref text, ref xRelStart, ref yRelStart, x, y, width, height, dtFlags, color, font);
         }
 
+        /// <summary>
+        /// Draws a single character to the screen
+        /// </summary>
+        /// <param name="c"> The character to draw.</param>
+        /// <param name="x"> The x-coordinate of the upper-left corner to draw to.</param>
+        /// <param name="y"> The y-coordinate of the upper-left corner to draw to.</param>
+        /// <param name="color"> The color to be used for the character.</param>
+        /// <param name="font"> The font to be used for the character.</param>
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        extern public void DrawChar(ushort c, int x, int y, Color color, Font font);
```

Comment:
Wouldn't the first parameter look better (obvious) being a char? 🤔

Reply:
It is 16bit in the underlying native code (I think to support unicode?) so I kept it as a short in the managed interface.

Reply:
OK. That makes sense let's leave it this way then.